docs/ops-recipe/harborDown.md

```diff
@@ -0,0 +1,67 @@
+---
+
+title: "Harbor Down"
+
+---
+
+This alert fires because a component of Harbor is not working as expected.
+
+First step would be to check if the pod is up and what the logs for the pod says:
+
+```
+
+> kubectl -n harbor-cluster logs harbor-$COMPONENT
+
+```
+You can also describe the pod:
+
+```
+
+> kubectl -n harbor-cluster describe pod harbor-$COMPONENT
+
+```
+
+Based on what the logs say try to figure out what is causing the error. Usually it will be the result of an improper configuration. Most configuration for the cluster is created in the [fullstack.yaml](https://github.com/goharbor/harbor-operator/blob/master/manifests/samples/full_stack.yaml). Below are some things to look out for if you cannot find the cause. 
+
+**Secrets**
+```
```

Comment:
Let's use codeblocks only for the actual executable code here, normal lists are fine.